frontend/src/app/components/user/profile/profile.component.html

```diff
@@ -2,8 +2,9 @@
   <div class="header-wrapper-wrapper">
     <div class="header-wrapper">
       <div class="header">
-        <span [routerLink]="['/']" class="username">{{userName}}</span>
-        <span class="id">user-id <span>#{{userId}}</span></span>
+          <span [routerLink]="['/']">
```

Comment:
I think it would be better if we do the navigation like on the product-information component with a button 'Home'